letsencrypt/plugins/manual_test.py

```diff
@@ -54,17 +54,10 @@ def test_perform(self, mock_raw_input, mock_verify, mock_stdout, mock_interactio
         resp = self.achalls[0].response(KEY)
         self.assertEqual([resp], self.auth.perform(self.achalls))
         self.assertEqual(1, mock_raw_input.call_count)
-        mock_verify.assert_called_with(
-            self.achalls[0].challb.chall, "foo.com", KEY.public_key(), 8080)
```

Comment:
I meant the variable mock_verify that you removed here. Some code for it is still on line 48, 50 and 51, right? Those can now be removed